apps/cli/src/commands/run.ts

```diff
@@ -86,7 +86,10 @@ export default class Run extends Command {
 
         // construct the path for Docker Compose files
         const composePath = (file: string) => path.join(binPath, "node", file);
-        const composeFiles = ["docker-compose-dev.yml"];
+        const composeFiles = [
+            "docker-compose-dev.yml",
+            "docker-compose-explorer.yml",
```

Comment:
Won't explorer be optional?

Reply:
What is your suggestion about how to enable/disable various services of the local env?

Reply:
We could split the `services:` definition in different `.yaml` files, and use some flag like `--disable-explorer`, composing the `composeFiles` array.

We need to remember that the order of compose files is important.

Reply:
I think having `--enable-<service>` and `--disable-<service>` flags will increase a lot the number of flags as we add services.

What about `--services=<service1>,<service2>`

Reply:
What services are we talking about? The ones defined at compose file, right? And not the ones inside the rollups-node.

Considering all services are enabled by default, we could have a `--disable-services=[...]`

Reply:
Put that in an Issue, please.

Reply:
Will do when the idea solidifies. For this issue I’ll code as including it by default.